src/asm/rgbasm.1

```diff
@@ -7,89 +7,187 @@
 .\"
 .Dd July 8, 2019
 .Dt RGBASM 1
-.Os RGBDS Manual
+.Os
 .Sh NAME
 .Nm rgbasm
 .Nd Game Boy assembler
 .Sh SYNOPSIS
-.Nm rgbasm
-.Op Fl EhLVvw
+.Nm
 .Op Fl b Ar chars
 .Op Fl D Ar name Ns Op = Ns Ar value
+.Op Fl E
 .Op Fl g Ar chars
+.Op Fl h
 .Op Fl i Ar path
+.Op Fl L
 .Op Fl M Ar dependfile
 .Op Fl o Ar outfile
 .Op Fl p Ar pad_value
 .Op Fl r Ar recursion_depth
-.Ar file
+.Op Fl V
+.Op Fl v
+.Op Fl w
+.Ar
 .Sh DESCRIPTION
 The
 .Nm
-program creates an object file from an assembly source file.
+program creates a RGB object file from an assembly source file.
 The input
 .Ar file
 can be a file path, or
 .Cm \-
 denoting
 .Cm stdin .
-Its arguments are as follows:
+.Pp
+Note that options can be abbreviated as long as the abbreviation is unambiguous:
+.Fl -verb
```

Comment:
`.Fl Fl verb`

Two reasons not to use `Fl -verb`:

* In typeset and HTML output, `groff` turns `Fl` into − while leaving - as -, resulting in “−-verb”.
* In `mandoc`’s semantic search, `Fl -verb` is found by searching for `-verb` while `Fl Fl verb` is found by searching for `verb`, which I think makes more intuitive sense.

Reply:
I didn't find any examples of mandoc with long options, and I didn't expect doubling `Fl` to work. Will fix as suggested.